### ISSUE TITLE
Enhancements to Content-Level Metatadata View/Edit

### ISSUE THREAD

**Issue by horsburgh** [hydroshare/hydroshare#3177]
Following up on #2813, I'm moving the remaining suggested work that was in the comments to the issue here as a new issue. The original work on #2813 is complete and deployed with the 1.19 release, so I'm creating this issue so we can close the other one. The following suggestions were made by @Lizabrazil 

* [ ] 1. When right clicking on a file, irrelevant options for content types should be disabled (greyed out) in the "Set Content Type" context menu.
* [ ] 2. Once the user clicks to "Add metadata" to a file, the right-side panel should automatically open the template to fill in the metadata.
* [ ] 3. The Add Metadata green icon should also appear in the right side panel so it is less hidden behind a right click. That is - in edit mode, when you select a file in the content area, the metadata area (right panel) should have a link and message to add metadata. Leave the right click add metadata there in the context menu as it is now.

**Comment by Maurier** [comment]
@horsburgh No. 2 and No. 3 have been implemented here and are waiting on code review:
https://github.com/hydroshare/hydroshare/pull/3112

**Comment by horsburgh** [comment]
@Maurier - thanks. I just verified that 1 is actually implemented. When I looked at the right click options the first time they didn't look like they were disabled, but they are. I'm going to close this one since 1 is done and 2 and 3 are in #3112.
